fluent/src/types.js

```diff
@@ -36,29 +36,45 @@ export class FluentType {
    * This method can use `Intl` formatters available through the `scope`
    * argument.
    *
-   * @param   {?object} scope
+   * @param   {Scope} scope
    * @returns {string}
    */
-  toString() {
+  toString(scope) { // eslint-disable-line no-unused-vars
     throw new Error("Subclasses of FluentType must implement toString.");
   }
 }
 
 export class FluentNone extends FluentType {
+  /**
+   * @param   {string} value - The fallback value of this FluentNone
+   * @returns {FluentType}
+   */
   constructor(value = "???") {
     super(value);
   }
 
+  /**
+   * @returns {string}
```

Comment:
esdoc generates a confusing doc string for this one, at least locally for me. This might need a full doc string and not just a return type?